e2e/cypress/integration/channel/user_to_admin_updates_manage_channel_members_modal_spec.js

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import users from '../../fixtures/users.json';
+
+const demoteToMember = (user) => {
+    cy.externalRequest({user: users.sysadmin, method: 'put', path: `users/${user.id}/roles`, data: {roles: 'system_user'}});
+};
+
+const promoteToSysAdmin = (user) => {
+    cy.externalRequest({user: users.sysadmin, method: 'put', path: `users/${user.id}/roles`, data: {roles: 'system_user system_admin'}});
+};
+
+describe('View Members modal', () => {
+    it('MM-20164 - Going from a Member to an Admin should update the modal', () => {
+        cy.apiLogin('user-1');
+        cy.apiGetMe().then((res) => {
+            // # Make user a regular member
+            demoteToMember(res.body);
+
+            // # Visit Town square and go to view members modal
+            cy.visit('/');
+            cy.get('#sidebarItem_town-square').click({force: true});
+            cy.get('#member_popover').click();
+            cy.findByTestId('membersModal').click();
+
+            // * Check to see if no drop down menu exists
+            cy.findAllByTestId('userListItemActions').then((el) => {
```

Comment:
Looks like this is not correct. `userListItemActions` can be found always.  Should probably check for its content to effectively test it.  Same case when promoting to admin.

Reply:
Agreed offline to just check text content when user is promoted.

Reply:
Done!